ui/src/app/pages/analysis/notebook-redirect/component.ts

```diff
@@ -191,9 +191,9 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
         // This will contain the Jupyter-local path to the localized notebook.
         if (!this.creating) {
           this.incrementProgress(Progress.Copying);
-          localizeObs = this.localizeNotebooks([this.notebookName],
+          localizeObs = this.localizeNotebooks([this.fullNotebookName],
```

Comment:
opt: minor cleanup - could just specify `this.fullNotebookName` at L286 rather than the append call. Would make these two API calls consistent